### PR TITLE
feat(babel-plugin-espower): async-await support

### PR DESCRIPTION
**Pull request by jamestalmage**
Call assertionVisitor with appropriate async/generator flags

[feat(espower): disambiguate between function calls and async/yield expressions by jamestalmage · Pull Request #23 · power-assert-js/espower](https://github.com/power-assert-js/espower/pull/23)

### CODE REVIEW

**Comment by twada** [comment]
@jamestalmage Thank you for your great contribution, again and again!

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "array-find": "^1.0.0",
     "escallmatch": "^1.4.2",
-    "espower": "^1.0.6",
+    "espower": "jamestalmage/espower#disambiguate-async-publish",
```
Comment (twada) [inline package.json]: Would you change this to espower 1.2.1 ?

File: test/test.js [hunk 2]
```diff
@@ -9,12 +9,16 @@ function testTransform (fixtureName, extraOptions) {
...
         }, extraOptions));
-        var actual = result.code;
-        var expected = fs.readFileSync(expectedFilepath).toString();
-        assert.equal(actual + '\n', expected);
+        var actual = result.code + '\n';
+        var expected = fs.readFileSync(expectedFilepath, 'utf8');
+        if (actual != expected) {
+            fs.writeFileSync(actualFilepath, actual);
+        }
+        assert.equal(actual, expected);
```
Comment (twada) [inline test/test.js]: Nice code for debugging!

**Comment by jamestalmage** [comment]
@twada done.

File: test/test.js (hunk 2 above)
Comment (jamestalmage) [inline test/test.js]: That, and I've got to imagine some of the generated code (especially the `async`/`generator` stuff) is going to change slightly over time with new versions of babel.

I pretty much just wrote a failing test and then manually inspected `actual` to verify what was there made sense (captures in the right place, `async` and `generator` context properties set correctly), and then copy/pasted.

Comment (twada) [inline test/test.js]: > I've got to imagine some of the generated code (especially the async/generator stuff) is going to change slightly over time with new versions of babel.

Yes, so useful tips :)

**Comment by twada** [comment]
@jamestalmage Looks good to me.
I'm going to cut a new 1.1.0 release soon.

**Comment by twada** [comment]
Just released [babel-plugin-espower 1.1.0](https://github.com/power-assert-js/babel-plugin-espower/blob/master/CHANGELOG.md#110-2015-11-06) :tada:

**Pull request merged**